docker-compose.yml

```diff
@@ -104,10 +151,16 @@ services:
       - "9090:9090"
     volumes:
       - spoke:/code
+    networks:
+      hubs_network:
 networks:
-  default:
+  hubs_network:
```

Comment:
The `default` applies the custom network (mozilla-hubs) to all the services. When you remove `default` you then have to specify the network for each service. That makes sense when you are divvying up the services between multiple networks, but we aren’t doing that. What’s the benefit in ditching the `default` here?

Reply:
IIRC it was required to assigning a static ip address.